make_single_file.php

```diff
@@ -59,7 +59,7 @@ function replace_wrl($file) {
 
     // @todo: this code will also include Inline wrl files that are commented out (if the files can be found), it might be hard to keep those out.
     $content = preg_replace_callback(
-        '/Inline\s?{\s+?url\s+?\["([^"]*)"\]\s+?}/i',
+        '/Inline\s*{\s*url\s+"([^"]*)"\s*}/ism',
```

Comment:
I believe the [] brackets are needed but should probably be opitional.

Reply:
Can you provide an example snippet of a multiline input? It is more urls or just a line break?